project/BuildHelper.scala

```diff
@@ -204,7 +204,7 @@ object BuildHelper {
     // autoAPIMappings := true,
     unusedCompileDependenciesFilter -= moduleFilter("org.scala-js", "scalajs-library"),
     Compile / fork := true,
-    Test / fork    := true, // set fork to `true` to improve log readability
+    Test / fork    := false, // set fork to `true` to improve log readability
```

Comment:
We will have to revert this when ready to merge.

Reply:
Yeah, I had to change that earlier to make debugging possible. From my notes (😅):

![obraz](https://user-images.githubusercontent.com/4235722/236467200-b5411384-7967-4b10-b2f2-e5fbba9736a3.png)